packages/schemas/src/lib/0-base/1-1-0.ts

```diff
@@ -0,0 +1,91 @@
+import type { JSONSchema7 } from 'json-schema';
+
+import { resolveGitHubPath } from '../helpers';
+import { credentialSubject } from '../schema-helpers';
+
+export const schema: JSONSchema7 = {
+  // draft-07 has the best support in ajv
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  $id: resolveGitHubPath('base/1-1-0.json'),
+
+  title: 'Base Verifiable Credential',
+  description:
+    'A generic Verifiable Credential schema for the Litentry Protocol.',
+
+  type: 'object',
+  required: [
+    '@context',
+    'issuer',
+    'issuanceDate',
+    'credentialSubject',
+    'proof',
+  ],
+  properties: {
+    '@context': {
+      type: 'array',
+      items: {
+        type: 'string',
+        format: 'uri',
+      },
+    },
+    id: {
+      type: 'string',
+    },
+    type: { type: 'array', items: { type: 'string' } },
+    issuer: {
+      type: 'object',
+      required: ['id', 'name', 'mrenclave'],
+      properties: {
```

Comment:
I must be missing something, but should we add `runtimeVersion` here? I'm a bit confused it is not part of the schema. The schema should validate it, if present.

Reply:
Good catch! I managed to clobber the definition when mass-editing the ts files :sweat: 
Ofc the tests didn't fail because it still matches :sweat_smile: 
Adding it back; thanks :pray: